src/plugins/formvalid/formvalid-server-fr.hbs

```diff
@@ -115,9 +112,36 @@
 						<legend class="required"><span class="field-name">Signature</span> <strong class="required">(Champ requis)</strong></legend>
 						<div class="checkbox"><label for="MainContent_wbCheckBoxList1_checkBoxList1_0"><input name="checkBoxList1" class="required_js valid" id="MainContent_wbCheckBoxList1_checkBoxList1_0" required="required" type="checkbox" checked="checked" value="checkBoxList1_0v"><span> <span class="field-name">Imprimer cette page</span></span> </label></div>
 						<div class="checkbox"><label for="MainContent_wbCheckBoxList1_checkBoxList1_1"><input name="checkBoxList1" id="MainContent_wbCheckBoxList1_checkBoxList1_1" type="checkbox" value="checkBoxList1_1v"><span> <span class="field-name">Veuillez m'envoyer de l'information complémentaire</span></span> </label></div>
-						<div class="checkbox required"><label for="MainContent_wbCheckBoxList1_checkBoxList1_2"><input name="checkBoxList1" id="MainContent_wbCheckBoxList1_checkBoxList1_2" aria-invalid="false" required="required" type="checkbox" value="checkBoxList1_2v"><span> <span class="field-name">Je suis en accord avec les directives</span><strong class="required"> (Champ requis)</strong></span><span class="label label-danger wb-server-error" id="MainContent_wbCheckBoxList1_checkBoxList1_2Cv"><strong><span class="prefix">Erreur&nbsp;4&nbsp;: </span>Ce champs est obligatoire</strong></span> </label></div>
+						<div class="checkbox required"><label for="MainContent_wbCheckBoxList1_checkBoxList1_2"><input name="checkBoxList1" id="MainContent_wbCheckBoxList1_checkBoxList1_2" aria-invalid="false" required="required" type="checkbox" value="checkBoxList1_2v"><span> <span class="field-name">Je suis en accord avec les directives</span><strong class="required"> (Champ requis)</strong></span><span class="label label-danger wb-server-error" id="MainContent_wbCheckBoxList1_checkBoxList1_2Cv">Ce champs est obligatoire</span></label></div>
 					</fieldset>
 
+					<fieldset class="form-inline chkbxrdio-grp">
+						<legend class="required"><span class="field-name">Statut de travail</span> <strong class="required">(Champ requis)</strong><span class="label label-danger wb-server-error" id="workError">Vous devez sélectionner Non pour cette question.</span></legend>
+						<label class="radio-inline">
+							<input class="input-validation-error" data-msg-required="Vous devez sélectionner Oui ou Non." data-rule-required="true" id="workNo" name="workStatus" type="radio" value="No" />
+							Non
+						</label>
+						<label class="radio-inline">
+							<input class="input-validation-error" id="workYes" checked="checked" name="workStatus" type="radio" value="Yes" />
+							Oui
+						</label>
+					</fieldset>
+
```

Comment:
Please add another example for `.checkbox-inline`.

```suggestion
<fieldset class="form-inline chkbxrdio-grp">
	<legend class="required"><span class="field-name">Animaux de compagnie</span> <strong class="required">(Champ requis)</strong><span class="label label-danger wb-server-error" id="petError">Un chevreuil n'est pas un animal de compagnie</span></legend>
	<label class="checkbox-inline"><input type="checkbox" checked="checked" name="petChoice" id="petDeer" required="required" /> Chevreuil</label>
	<label class="checkbox-inline"><input type="checkbox" name="petChoice" id="petCat" /> Chat</label>
	<label class="checkbox-inline"><input type="checkbox" name="petChoice" id="petDog" /> Chien</label>
</fieldset>
```

Reply:
Example added.